TESTING.adoc

```diff
@@ -0,0 +1,91 @@
+[[addons-tests]]
```

Comment:
@robin-owen Please take a look on this file when you have time. I would like to also ask if it feels understandable - as I am quite familiar with Minishift integration tests and Gherkin I feel my opinion here might be affected. Thanks :)